### DISCUSSION TITLE
Toast click action with -Appid

### DISCUSSION THREAD

**Discussion by JerryTsaiTPE** [Windos/BurntToast#217] [Q&A]
I try to replace the "Windows PowerShell" on top of the toast.
The way I found is to use [-Appid] to replace "Windows PowerShell" with other apps in the system. (Example: "Messaging")  
When I click on the Toast(not on the button), it always pops up Messaging app. How can I change this click action to open the same hyperlink(or do nothing) and keep the Messaging app icon and title? 

Or, is another way to replace the "Windows PowerShell" and the PowerShell icon?
I'm using the Win11 OS.

Thanks.

`
$Button1  = New-BTButton -Content 'CLICK HERE' -Arguments 'https://www.google.com/intl/en_us/chrome/'

$splat = @{
Text    = 'Check for Chrome update', 'Sent from Jerry'
Sound   = 'IM'
AppLogo = 'C:\icon\GoogleChrome.png'
Button  = $Button1
}

New-BurntToastNotification @Splat -appid 'Microsoft.Messaging_8wekyb3d8bbwe!x27e26f40ye031y48a6yb130yd1f20388991ax'
`

**Comment by Windos** [comment]
Hello, and thank you for your contribution to BurntToast.

Today marks the official release of [BurntToast v1.0.0](https://github.com/Windos/BurntToast/releases/tag/v1.0.0). As part of this major milestone and to ensure the project starts its next chapter with a clean slate, I am declaring "issue bankruptcy."

This decision is explained in more detail in the official announcement post: [Ten Years Toasting: BurntToast Hits the Big v1!](https://toastit.dev/2025/07/22/burnttoast-v1/)

As a result, I am closing all outstanding issues and pull requests (and discussions), including this one. Please do not interpret this as a lack of interest in the problem, suggestion, or topic you've raised. This is a necessary step to clear a very old backlog and focus development efforts on the current state of the module.

**If this topic is still relevant to you:**

1.  Please **open a new discussion**. You are welcome to link back to this one for historical context.

I hope this reset will allow me to be more responsive to community feedback going forward.

Thank you for your understanding and for being part of the BurntToast journey.
